pulsar-broker/src/test/java/org/apache/pulsar/client/api/SimpleProducerConsumerTest.java

```diff
@@ -125,13 +126,35 @@ public class SimpleProducerConsumerTest extends ProducerConsumerBase {
     private static final int RECEIVE_TIMEOUT_SHORT_MILLIS = 200 * TIMEOUT_MULTIPLIER;
     private static final int RECEIVE_TIMEOUT_MEDIUM_MILLIS = 1000 * TIMEOUT_MULTIPLIER;
 
-    @BeforeMethod(alwaysRun = true)
+    @BeforeClass
     @Override
     protected void setup() throws Exception {
         super.internalSetup();
         super.producerBaseSetup();
     }
 
+    @AfterMethod(alwaysRun = true)
+    public void rest() throws Exception {
+        pulsar.getConfiguration().setForceDeleteTenantAllowed(true);
+        pulsar.getConfiguration().setForceDeleteNamespaceAllowed(true);
+
+        for (String tenant : admin.tenants().getTenants()) {
+            for (String namespace : admin.namespaces().getNamespaces(tenant)) {
+                admin.namespaces().deleteNamespace(namespace, true);
```

Comment:
Suggest to use `deleteNamespaceGraceFully`, because `delete namespace` is not a stable cmd, see https://github.com/apache/pulsar/pull/17157. 

Note that `deleteNamespaceGraceFully` runs slower than `delete namespace` and is unsuitable for all scenarios.

Reply:
Ok, thanks.

Reply:
> Suggest to use `deleteNamespaceGraceFully`, because `delete namespace` is not a stable cmd, see #17157.
> 
> Note that `deleteNamespaceGraceFully` runs slower than `delete namespace` and is unsuitable for all scenarios.

in my CI test, There are two failures related to these changes:
Time elapsed failures while excute `deleteNamespaceGraceFully `
see the checks in https://github.com/apache/pulsar/actions/runs/3242148385/jobs/5326408439
could i skip this test temporary？

Reply:
@poorbarcode hi
my CI check always fails in Pulsar CI Flaky / Flaky tests suite  . seem releted to run `deleteNamespaceGraceFully`
cloud you please help me confirm the reason? thx
https://github.com/apache/pulsar/actions/runs/3242148385/jobs/5359655987

Reply:
@ethqunzhong I see, could you `rebase master` and see if it reproduces?

Reply:
> 

@poorbarcode thx your reply.